src/Tools/BuildBoss/ProjectCheckerUtil.cs

```diff
@@ -62,6 +62,7 @@ public bool Check(TextWriter textWriter)
                 allGood &= CheckProjectReferences(textWriter);
                 allGood &= CheckPackageReferences(textWriter);
                 allGood &= CheckDeploymentSettings(textWriter);
+                // allGood &= CheckTargetFrameworks(textWriter);
```

Comment:
@tmat this PR is still "for personal review". I wanted to verify some stuff with Jenkins and commented out this block temporarily to not get blocked by it.